documentation/modules/adding-source-provider.adoc

```diff
@@ -26,37 +26,174 @@ You can add {a-rhv} source provider by using the {ocp} web console.
+* *Authentication type*: Choose one of the following methods of authentication and supply the information related to your choice. For example, if you choose *Application credential ID* as the authentication type, the *Application credential ID* and the *Application credential secret* fields become active, and you need to supply the ID and the secret.
+
+** *Application credential ID*
+*** *Application credential ID*: {osp} application credential ID
+*** *Application credential secret*: {osp} https://github.com/kubev2v/forklift-documentation/pull/402pplication credential `Secret`
+
+** *Application credential name*
+*** *Application credential name*: {osp} application credential name
+*** *Application credential secret*: : {osp} application credential `Secret`
+*** *Username*: {osp} username
+*** *Domain*: {osp} domain name
+
+** *Token with user ID*
+*** *Token*: {osp} token
+*** *User ID*: {osp} user ID
+*** *Project ID*:  {osp} project ID
+
+** *Token with user Name*
+*** *Token*: {osp} token
+*** *Username*: {osp} username
+*** *Project*: {osp} project
+*** *Domain name*: {osp} domain name
+
+** *Password*
+*** *Username*: {osp} username
+*** *Password*: {osp} password
+*** *Project*: {osp} project
+*** *Domain*: {osp} domain name
+
+. Choose one of the following options for validating CA certificates:
+
+** *Skip certificate validation* : Migrate without validating a CA certificate.
+** *Use the system CA certificates*: Migrate after validating the system CA certificates.
+** *Use a custom CA certificate*: Migrate after validating a custom CA certificate.
 
-. To allow a migration without validating the provider's CA certificate, select the *Skip certificate validation* check box. By default, the checkbox is cleared, meaning that the certificate will be validated.
-. If you did not select *skip certificate validation*, the *CA certificate* field is visible. Drag the CA certificate to the text box or browse for it and click *Select*. Use the {manager} CA certificate or {manager} Apache CA certificate if the {manager} CA certificate was replaced by a third-party certificate on the Apache server. If you did select the check box, the *CA certificate* text box is not visible.
+.. To skip certificate validation, select the *Skip certificate validation* check box.
+.. To validate the system CA certificates, leave the *Skip certificate validation* check box cleared and leave the *CA certificate* text box empty.
+.. To validate a custom CA certificate, leave the *Skip certificate validation* check box cleared and _either_ drag the CA certificate to the text box _or_ browse for it and click *Select*.
+endif::[]
+ifdef::cnv[]
+. Click *{virt}*.
+. Specify the following fields:
+
+* *Provider resource name*: Name of the source provider
+* *URL*: Endpoint of the API server
+* *Service account bearer token*: Token for a service account that is assigned `cluster-admin` permissions
```

Comment:
I see we write "You must have an OpenShift Virtualization service account token with cluster-admin privileges." above so maybe it's better to phrase it as
```suggestion
* *Service account bearer token*: Token for a service account with `cluster-admin` privileges
```
but that's a phrasing thingie, I'll let you to decide on

Reply:
Done.